src/InOneWeekend/main.cc

```diff
@@ -20,16 +20,19 @@
 
 
 void random_spheres(scene& scene_desc) {
-    scene_desc.image_width       = 1200;
-    scene_desc.aspect_ratio      = 16.0 / 9.0;
-    scene_desc.samples_per_pixel = 10;
+    scene_desc.cam.lookfrom = vec3(13,2,3);
+    scene_desc.cam.lookat   = vec3(0,0,0);
+    scene_desc.cam.vup      = vec3(0,1,0);
+    scene_desc.cam.vfov     = 20;
+
+    scene_desc.cam.aspect_ratio = 16.0 / 9.0;
+    scene_desc.cam.image_width  = 1200;
 
-    scene_desc.cam.vfov       = 20;
-    scene_desc.cam.focus_dist = 10.0;
-    scene_desc.cam.aperture   = 0.1;
-    scene_desc.cam.lookfrom   = vec3(13,2,3);
-    scene_desc.cam.lookat     = vec3(0,0,0);
-    scene_desc.cam.vup        = vec3(0,1,0);
+    scene_desc.cam.defocus_diameter = 0.1;
+    scene_desc.cam.focus_dist       = 10;
+
+    scene_desc.samples_per_pixel = 10;
```

Comment:
My mental model has samples_per_pixel in camera rather than scene, but I'm happy with either

Reply:
Hmm. 

I'm trying to think what be the more consistent mental model for max_depth would be.

If we moved samples_per_pixel over to camera would it make sense to keep max_depth in the scene class or in the camera class....

The number of bounces is pretty much solely dependent on geo+lighting, which is very much contained in scene. But it _feels_ like it would be coupled with samples_per_pixel.

I'm fine either way. Just things to consider